### PR TITLE
Fix messages

### PR DESCRIPTION
**Pull request by victorwss**
Pull request for #10811 and #10812.

Feel free to comment if there is some doubt or suggestion and the descriptions of #10811 and #10812 aren't enough.

I also think about some further changes, but this pull request is already a large one, so I will save them for later.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/Checker.java [hunk 1]
```diff
@@ -318,6 +316,7 @@ private void processFiles(List<File> files) throws CheckstyleException {
      * @param file a file to process.
      * @return a sorted set of violations to be logged.
      * @throws CheckstyleException if error condition within Checkstyle occurs.
+     * @return The set of violations found
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/Checker.java:319]: There is already a return tag 2 lines up, so this change isn't right.

**Review by rnveach — CHANGES_REQUESTED** [review]
@victorwss I recommend splitting this up into multiple PRs. Things can be merged a lot faster the smaller the change is that admins need to review.

We have no one on hand to review translations, so your input is greatly appreciated to help fix those.

File: src/main/java/com/puppycrawl/tools/checkstyle/Checker.java (hunk 1 above)
Comment (victorwss) [reply src/main/java/com/puppycrawl/tools/checkstyle/Checker.java:319]: Pfft. This is a leftover from some refactor that I late undid.

**Comment by victorwss** [comment]
@rnveach Do you think that it is better to cancel this big fat PR and redo it in as a bunch of newly created baby-steps PRs? Or should I rework this PR somehow with further commits?

Also, if I should redo from start as baby-steps, can the translations go into a single step or should I work in baby steps with them even if I revisit the same translation more than once? Or should I do a PR for each language?

This PR has a lot of files, but they are divided in four categories: 95 properties files; 30 resource java files full of violations where I just removed some hardcoded error messages; 9 files containg unit tests that were changed; 12 actual files changed and 1 added in the checkstyle production code. It is a lot, but still quite big, if you just pass an quick eye in the 95+30 and focus in the 9+12+1.

Also, should I divide the issues #10811 and #10812 into a bunch of small issues? Or just the PR need to be divided?

File: src/main/java/com/puppycrawl/tools/checkstyle/Checker.java (hunk 1 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/Checker.java]: We have a Javadoc linter in CI. It should be configured to fail on warnings.

**Comment by rnveach** [comment]
> created baby-steps PRs

Yes.

> can the translations go into a single step

Yes. I noticed some english translations were changed grammatically, which is the one ones we can really probably have a discussion on. If possible, can you split between fixing up the translation and redoing the whole message?

> divide the issues ... into a bunch of small issues? Or just the PR need to be divided

I am leaning more to splitting the PRs, but I think the one issue may need some division as well at just a high level. I haven't read everything you identified, but #10812 seems a bit bloated and may make it hard to tell if the full work is completed or not. So you know, each issue mapped to and fixed is put on the release notes so everything on 1 issue will just be 1 line in the release notes. That does not make it clear to the release notes that a lot of fixes were made and what would exactly impact them.

**Comment by romani** [comment]
@victorwss , I really appreciate your help, and sorry that such PR did come to my attention sooner.

Please split this huge PR is few multiple. It is always good to keep PR in consumable size with clear single goal.

**Comment by Rahulkhinchi03** [comment]
@victorwss please let me know if I can help you in this PR?

**Comment by rnveach** [comment]
@victorwss ping

**Comment by nick-mancuso** [comment]
We lost contact with author, anybody is welcome to break changes in this PR into smaller PRs to help solve https://github.com/checkstyle/checkstyle/issues/10811 and https://github.com/checkstyle/checkstyle/issues/10812.

**Comment by victorwss** [comment]
@rvneach Sorry for losing contact, I had a lot of health issues in the meantime. I have a lot of things to fix in my life, maybe I will try to engage with checkstyle again in the future.

**Pull request closed without merging**